proto/regen/ecocredit/basket/v1/state.proto

```diff
@@ -48,7 +48,7 @@ message Basket {
   // basket settings.
   //
   // Since Revision 1
-  string curator = 8;
+  bytes curator = 8;
```

Comment:
Here's an example of a revision comment since we added this field since the original v1 proto version released as part of Regen Ledger v3.0.